### ISSUE TITLE
Encrypted sqlite db will not open

### ISSUE THREAD

**Issue by carlfitzsimmons** [sqlitebrowser/sqlitebrowser#1241]
#### I'm opening this issue because:

- [ ] DB4S is crashing
- [ ] DB4S has a bug
- [ ] DB4S needs a feature
- [x] DB4S has another problem

#### I'm using DB4S on:

- [ ] Windows: ( _version:_ ___ )
- [ ] Linux: ( _distro:_ ___ )
- [x] Mac OS: ( _version:10.13.1 )
- [ ] Other: ___

#### I'm using DB4S version:

- [x] 3.10.1
- [ ] 3.10.0
- [ ] 3.9.1
- [ ] Other: ___

#### I have also:

- [ ] Tried out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [x] Searched for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by chrisjlocke** [comment]
> using SQLite version: 3.19.3 with an encryption key

SQLite doesn't natively come with encryption.  Do you mean using SEE (https://www.sqlite.org/see) ?
If so, then SQLCipher (the library used by DB4S) isn't compatible with SEE.

```Cipher is not at all compatible at a database level with SEE. That is to say, regardless of what cipher is used, you can't open a SEE database with SQLCipher (or the other way).```

Your only option is to try to use SEE to convert the database to a standard non-encrypted database, then use SQLCipher to convert it to a SQLCipher database.

**Comment by justinclift** [comment]
Just went looking for a way to show hidden files by default.  Doing a search online shows up [other Qt programs](https://github.com/lxde/pcmanfm-qt/commit/4ac1b8dcf5298b50e46738ced78f3acab3dcd345) using a function called `showHidden()` and `setShowHidden()`.  I'm not sure if they're Qt functions (likely), or something implemented elsewhere inside that program.

Going to the Qt documentation website... the ability to search through the docs seems to have been removed.  (WTF?)

Doing a quick search for "showhidden" in the help section of Qt Creator leads to this:

&nbsp; &nbsp; http://doc.qt.io/qt-5/qml-qt-labs-folderlistmodel-folderlistmodel.html#showHidden-prop

So, it might be workable. :smile:

**Comment by mgrojo** [comment]
There is usually a built-in option in the standard file dialog for showing hidden files. For example, in my scenario (Ubuntu 14.06) it is shown in the context menu appearing with a right-click over the file listing.

**Comment by iKlsR** [comment]
Do we want this on by default?

**Comment by justinclift** [comment]
Hmmmm, no preference either way personally. :smile:

**Comment by bich30** [comment]
Yes, Of course!!

**Comment by Klojum** [comment]
Yes please, I would prefer an 'Always on' situation, also for the browsing part as I work with hidden folders in which non-hidden sqlite db files reside. Working in Ubuntu Linux, but I'm sure people with MacOS have the same experience.

**Comment by scottfurry** [comment]
@Klojum - This is a function of the underlying Qt libraries and the operating system. If you right-click when opening a file, you should see the option to "show hidden files".
